pages/docs/guides/resend-webhook-events.mdx

```diff
@@ -0,0 +1,325 @@
+export const description = 'Set up Resend webhooks with Inngest and use Resend events within Inngest functions.'
+
+# Integrate email events with Resend webhooks
+
+[Resend webhooks](https://resend.com/docs/dashboard/webhooks/introduction) can be used to build functionality into your application based on changes in the email status. In this guide, you will learn:
+
+- What webhook events are offered by Resend.
+- How to set up Inngest to receive Resend webhook events.
+- How to define Inngest functions in your application using Resend events.
+- How to build a dynamic drip marketing campaign which responds to a user's behavior.
+
+To follow this guide, you need a [Resend](https://resend.com/) and [Inngest](https://www.inngest.com/) accounts and have Inngest [set up](https://www.inngest.com/docs/quick-start) in your codebase.
```

Comment:
Can we change the Inngest links to relative URLs for broken link checking and whatnot?